packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -142,20 +146,51 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
         return "";
     }
 
-    public async createBlob(file: Uint8Array): Promise<api.ICreateBlobResponse> {
-        this.checkSnapshotUrl();
+    public async createBlob(file: IsoBuffer): Promise<api.ICreateBlobResponse> {
```

Comment:
file is Uint8Array on interface, why is it IsoBuffer here?

Reply:
The interface can't use `IsoBuffer` since it violates layer-check.

Reply:
Then should it be Uint8Array here then? Interface basically says that client is free to provide Uint8Array, and driver can assume it to have any members that are only on IsoBuffer.
Maybe other way to say it - it needs to accept Uint8Array  and wrap it with IsoBuffer if that's the direction we want to go.